tools/test-setup-after-env.js

```diff
@@ -1 +1,3 @@
 import '@testing-library/jest-dom'
+
+jest.mock('scheduler', () => require('scheduler/unstable_mock')) // eslint-disable-line
```

Comment:
Can we add a comment on what this is for? I'm guessing it's related to the new `createRoot` call in React 18, but it might be nice to add some context! (Plus the specific rule that's being disabled here):

`// eslint-disable-next-line some-rule -- Scheduler needs to be mocked for React 18`

Reply:
yep, this makes a lot of sense, thanks for the feedback

Reply:
yep, this makes a lot of sense, thanks for the feedback - done! @brandongregoryscott